### PR TITLE
DBZ-4907 Increment Commit SCN when any COMMIT is detected

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-4907

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Still for the Final, @Naros?

File: debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogMinerQueryBuilderTest.java [hunk 1]
```diff
@@ -280,6 +282,7 @@ private String resolveLogMineryContentQueryFromTemplate(OracleConnectorConfig co
         query = query.replace("${operationCodes}", config.isLobEnabled() ? OPERATION_CODES_LOB_ENABLED : OPERATION_CODES_LOB_DISABLED);
         query = query.replace("${schemaPredicate}", schemaReplacement == null ? "" : schemaReplacement);
         query = query.replace("${tablePredicate}", tableReplacement == null ? "" : tableReplacement);
+        System.out.println(query);
```
Comment (jpechane) [inline debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogMinerQueryBuilderTest.java:285]: This should not be here.

**Review by jpechane — COMMENTED** [review]
@Naros This looks good. I left only one nitpick comment,
I just wonder, should not this change be reflected in any of comments?

File: debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogMinerQueryBuilderTest.java (hunk 1 above)
Comment (gunnarmorling) [reply debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogMinerQueryBuilderTest.java:285]: Don't we have a check for forbidden APIs in place yet? If not, I'd suggest we add https://github.com/policeman-tools/forbidden-apis for 2.0.

Comment (jpechane) [reply debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogMinerQueryBuilderTest.java:285]: @gunnarmorling The checkstyle catches this, but it is error for main code only, warning for tests.

Comment (Naros) [reply debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogMinerQueryBuilderTest.java:285]: Grrr, doh!  Thanks for the quick spot that I missed.

**Pull request merged**